draft-ietf-rats-architecture.md

```diff
@@ -630,6 +630,36 @@ plans to support in the TEEP architecture {{-teep-arch}}.
 ~~~~
 {: #combination title="Example Combination"}
 
+# Roles and Entities
+
+An entity in the RATS architecture includes at least one of the roles defined
+in this document. As a result, the entity can participate as a constituent of
+the RATS architecture. Additionally, an entity can aggregate more than one
+role into itself. These collapsed roles combine the duties of multiple roles.
+In these cases, interaction between these roles do not necessarily use the
+Internet Protocol. They can, e.g., using a loopback device or other IP-based
+communication between separate environments, but they do not have to.
+Alternative channels to convey conceptual messages include sockets, GPIO
+interfaces, local busses, or hypervisors. This conveyance is sometime
+referred to as "local conveyance". This type of conveyance is typically found
+in Composite Devices. Most importantly, these conveyance methods are
+out-of-scope of RATS, but they are presumed to exist in order to convey
+conceptual messages appropriately between roles.
+
+The following paragraph elaborates on an exemplary usage scenario:
+
+An entity that both connects to a wide-area
+network and to a system bus taking on both the Attester and Verifier roles.
+As a system bus entity, a Verifier consumes Evidence from other devices
```

Comment:
Is this referring to a Composite Device example?  If so, move it to that section, or maybe it's already covered there.

Reply:
Could be, but that's not the point.